stl/src/xonce2.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// __std_execute_once_begin & __std_execute_once_end functions
+
+#include "xcall_once.h"

```

Comment:
This header lives in `stl/inc` so it should be included with angle brackets. (Note that clang-format will then re-sort it.)